executor/modes/local/command_handler.py

```
@@ -0,0 +1,214 @@
+
+    async def _terminate_process(self, process: asyncio.subprocess.Process) -> None:
+        if process.returncode is not None:
+            return
+
+        try:
+            if os.name != "nt":
+                os.killpg(process.pid, signal.SIGTERM)
+            else:
+                process.terminate()
+            await asyncio.wait_for(process.wait(), timeout=2)
+        except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)